opsimulator/deposits.go

```diff
@@ -22,9 +22,20 @@ type depositTxSubscription struct {
 	doneCh          chan struct{}
 }
 
+type DepositChannels struct {
+	DepositTxCh chan<- *types.DepositTx
+	LogCh       chan<- types.Log
+}
```

Comment:
We should ideally just remove this file altogether, and in the indexer logic just subscribe to the log.

The indexer can subscribe to the log, then manually convert it to a deposit tx.